yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -236,6 +234,31 @@ func generateOrGetStreamIDForYugabyteCDCClient(config *dbzm.Config) error {
 	return nil
 }
 
+func fetchOrRetrieveColToSeqMap(msr *metadb.MigrationStatusRecord, tableList []sqlname.NameTuple) map[string]string {
+	var storedColToSeqMap map[string]string
+	//fetching the stored one in the MSR
+	switch exporterRole {
+	case SOURCE_DB_EXPORTER_ROLE:
+		storedColToSeqMap = msr.SourceColumnToSequenceMapping
+	case TARGET_DB_EXPORTER_FB_ROLE, TARGET_DB_EXPORTER_FF_ROLE:
+		storedColToSeqMap = msr.TargetColumnToSequenceMapping
+	}
+	if len(lo.Keys(storedColToSeqMap)) > 0 {
```

Comment:
what if the source actually does not have any sequences? this will be empty and we will go to source? 
Can we probably do a NULL check instead of len(keys)?